### ISSUE TITLE
CellPose in Google Colab Notebook not able to display 3D segmentation Results

### ISSUE THREAD

**Issue by ss8319** [MouseLand/cellpose#471]
Hi.
Background:
I am working on a project to perform 3D segmentation using cellpose on stem cell images obtained from confocal microscopy.

Problem
I am using this Colab Notebook. ([Google Colab 1](https://colab.research.google.com/github/MouseLand/cellpose/blob/master/notebooks/run_cellpose_GPU.ipynb))

According to the notebook there are 2 ways to perform 3D segmentation. First way: computes flows from 2D slices and combines into 3D flows to create masks. Second way: computes masks in 2D slices and stitches masks in 3D based on mask overlap.

I am using the 1st way. However in its subsequent code to display 3D masks from flows. I keep running into this error.

Error
index 758 is out of bounds for axis 1 with size 3 ( If I remembered correctly, 758 is the correct number)

I think its worth noting that my data file named stack.tif (RGB) has the dimensions [5,1024,1024,3] where [nplanes x nY x nX x channels]. From the cellpose documentation([Inputs — cellpose 0.7.2 documentation](https://cellpose.readthedocs.io/en/latest/inputs.html)), it seems like multiplane images should be of shape nplanes x channels x nY x nX or as nplanes x nY x nX. I think this might be a source of error. Otherwise how can I edit the code to finish off with the 3D segmentation(produce 3D masks) and visualize the results of 3D segmentation.

**Comment by carsen-stringer** [comment]
cellpose has options for `channel_axis` and `z_axis`, please set `channel_axis=3` and `z_axis=0` in `model.eval` e.g.:
```
from cellpose.models import CellposeModel
model = CellposeModel()
masks = model.eval(img, channel_axis=3, z_axis=0, channels=[1,2], do_3D=True, diameter=30)[0]
```
